src/components/welcome/MyAccounts/QueueActions.tsx

```diff
@@ -44,27 +44,25 @@ const QueueActions = ({
   }
 
   return (
-    <Box width="100%">
```

Comment:
I think we need the Box as a wrapper otherwise the Chips are not below the address:
<img width="797" alt="Screenshot 2024-10-29 at 10 04 12" src="https://github.com/user-attachments/assets/7e4622e2-cb79-4451-ab32-3bea0d451f72">